README.md

```diff
@@ -32,6 +32,7 @@ curl "http://npmsearch.com/query?q=dom&fields=name"
 * __scripts__
 * __times__
 * __version__
+* __score__
 * __rating__ - computed rating as per [bin/rating.js](bin/rating.js)
```

Comment:
@tmpvar since `rating` is no longer computed this way, should we remove the file altogether (`bin/rating.js`) or remove this explanation at least ?